### PR TITLE
Add Azure Policy Add-on labelSelector image to VHD

### PR DESCRIPTION
**Pull request by RamyasreeChakka**
Add Azure Policy Add-on labelSelector image to VHD.

### CODE REVIEW

**Comment by RamyasreeChakka** [comment]
@xuto2 @robbiezhang @ruchikaguptaa - Please review when you get a chance.

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -540,6 +540,7 @@ mcr.microsoft.com/oss/kubernetes/ingress/nginx-ingress-controller:0.19.0
 mcr.microsoft.com/oss/virtual-kubelet/virtual-kubelet:1.2.1.1
 mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod:prod_20200901.1
```
Comment (robbiezhang) [inline vhdbuilder/packer/install-dependencies.sh:541]: can we remove this?

**Review by robbiezhang — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (RamyasreeChakka) [reply vhdbuilder/packer/install-dependencies.sh:541]: prod_20200901.1 image is still in use. prod_20200923.1 image is being deployed in AKS 20201012 bits.

**Review by ruchikaguptaa — COMMENTED** [review]
lgtm

**Pull request merged**